Kickstarter-iOS/Views/Cells/PledgeDescriptionCell.swift

```diff
@@ -51,9 +51,8 @@ final class PledgeDescriptionCell: UITableViewCell, ValueCell {
     self.configureStackView()
 
     NSLayoutConstraint.activate([
-      self.containerImageView.widthAnchor.constraint(equalToConstant: Layout.Container.width),
```

Comment:
Setting width constraint on the container was overriding image constraints....we actually don't need to set container's constraints since it will stretch based on the image view.